django/db/models/base.py

```diff
@@ -461,7 +463,13 @@ def __init__(self, db=None):
 class Model(six.with_metaclass(ModelBase)):
 
     def __init__(self, *args, **kwargs):
-        signals.pre_init.send(sender=self.__class__, args=args, kwargs=kwargs)
+        # Alias some things as function locals to avoid repeat global lookups
```

Comment:
Will chop

The `meta = meta` optimization is done in some other methods in this file in e.g. `_save_table`